Makefile

```diff
@@ -35,10 +35,10 @@ php-cs-fix:																		## run cs fixer
 	PHP_CS_FIXER_FUTURE_MODE=1 $(PHP_CS_FIXER_BIN) fix --allow-risky=yes
 
 phpstan:
-	$(PHPSTAN_BIN) analyse
+	$(PHPSTAN_BIN) analyze
```

Comment:
This is a phpstan command, that uses the `analyse` spelling as seen in the [docs](https://phpstan.org/user-guide/command-line-usage#analysing-code) and should not be changed

Reply:
And yes, they have an alias too, but it could be dropped at any moment. `analyse` is the supported spelling.

Reply:
I agree. The renaming should be limited to deptrac, not other tools like phpstan or psalm